### PR TITLE
few things for rails5

### PR DESCRIPTION
**Pull request by mathieujobin**

### CODE REVIEW

File: lib/oauthenticator/faraday_signer.rb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-require 'oauthenticator'
```
Comment (mathieujobin) [inline lib/oauthenticator/faraday_signer.rb]: complains about a recursive require

**Comment by mathieujobin** [comment]
would be great if you could merge this @notEthan 

thanks

File: lib/oauthenticator/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # OAuthenticator 
 module OAuthenticator
   # OAuthenticator::VERSION
-  VERSION = "1.3.1"
+  VERSION = "1.3.4"
```
Comment (notEthan) [inline lib/oauthenticator/version.rb]: typo? commit says 1.3.2 which would be the logical next number here

Comment (notEthan) [inline lib/oauthenticator/version.rb]: I will merge the previous commit. it is preferable to me to manage versions outside of pull requests.

Comment (notEthan) [inline lib/oauthenticator/version.rb]: actually that will mess up the pr. can you just push up the two commits for this pr?

**Comment by notEthan** [comment]
bump for travis

**Comment by notEthan** [comment]
oops, I meant to do that on the other pr ... (the one with the travis yml)

File: lib/oauthenticator/version.rb (hunk 2 above)
Comment (mathieujobin) [inline lib/oauthenticator/version.rb]: confusion between 1.3.2 and 1.3.4 came from version in venuenext repo and this one...
I must have amend my commit message but forgot to git add the number change.

anyway, I will remove that commit from this PR, so you can manage the versions outside of PR like you ask

**Comment by mathieujobin** [comment]
done

File: oauthenticator.gemspec [hunk 3]
```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   ]
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rack', '~> 1.4'
+  spec.add_runtime_dependency 'rack', '>= 1.4', '< 2.1'
```
Comment (notEthan) [inline oauthenticator.gemspec]: hm, rack 2 isn't even out, hard to be sure of compatibility.
also by semantic versioning if this is compatible with 2.0 then it should be compatible < 3. in practice who knows.
this restriction is fine for now, then.

**Pull request merged**